src/test/librbd/deep_copy/test_mock_ImageCopyRequest.cc

```diff
@@ -221,6 +221,9 @@ TEST_F(TestMockDeepCopyImageCopyRequest, SimpleImage) {
 }
 
 TEST_F(TestMockDeepCopyImageCopyRequest, OutOfOrder) {
+  CephContext* cct = reinterpret_cast<CephContext*>(_rados.cct());
+  REQUIRE(cct->_conf->get_val<int64_t>("rbd_concurrent_management_ops") >= 10);
```

Comment:
Since I don't think we have any teuthology tests that would change this config parameter, this test would effectively never be run. Perhaps this test should store the current setting, update it, and then restore it prior to finishing.